appData/src/gb/include/gbs_types.h

```diff
@@ -1,9 +1,7 @@
 #ifndef GBS_TYPES_H
 #define GBS_TYPES_H
 
-#ifdef CGB
-    #include <gb/cgb.h>
-#endif
+#include <gb/cgb.h>
```

Comment:
Is this ifdef removal needed for the feature?

Reply:
Ooops not sure why it's here